### DISCUSSION TITLE
About training

### DISCUSSION THREAD

**Discussion by kaian9804** [pykt-team/pykt-toolkit#83] [Q&A]
Hello, I am training the DKT model with pykt. In the pykt document, I found that the trained model will be saved. May I know the place where the model saved by default? I can't find the dir like "save_model"... Also, what should be defined by the "--dataset_name"? Is that the dataframe that I used for training? Thanks!

**Comment by meta-tabchen** [comment]
Thanks for your attention to our project. 

1. The root of the save path is the "examples" directory, the subfolder depends on the input, you can find the subfolder in the log, here is one example:
![image](https://user-images.githubusercontent.com/19347690/209510001-b9f97c81-b1ad-418a-8dc9-46b3e813c0bd.png)

2. Yes, "--dataset_name" is the dataset name you want to train.

**Reply by kaian9804** [reply]
Thanks for your reply.
However, I face bugs while training, it always appear the KeyError with "test_file". It cause I didn't have a completed model yet. I also put this issue in another discussion, do you know what is happening? Thanks!
![no_test_file](https://user-images.githubusercontent.com/34164281/209533663-386b7df4-1b9e-4200-83e5-200593109fc4.png)

**Reply by meta-tabchen** [reply]
You can try to clone the codes on the dev branch, and `pip install -e .`  to install it. If the problem still exists, please let us know.

**Reply by kaian9804** [reply]
> You can try to clone the codes on the dev branch, and `pip install -e .` to install it. If the problem still exists, please let us know.
It is not able to install "pip install -e ." Do you have any idea with this error? Thanks.

The error msg is as follow:
Looking in indexes: https://pypi.org/simple, https://us-python.pkg.dev/colab-wheels/public/simple/
Obtaining file:///content
ERROR: file:///content does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
